chart.go

```diff
@@ -122,6 +122,8 @@ type Range struct {
 	InvNorm     func(float64) float64 // Inverse of Norm()
 	Data2Screen func(float64) int     // Function to map data value to screen position
 	Screen2Data func(int) float64     // Inverse of Data2Screen
+
+	AlignTrans bool // alignment translation of the axis: false: axis location left/bottom, true: axis location right/top
```

Comment:
AlignTrans is not a clear name. I would prefer something like SwitchSide or AlternateSide or something else but it is not just a translation, and not really an alignment.

For the comment: Start with a capital letter.

Maybe

    SwitchSide bool // Switch location of axis to top / right if true.